### PR TITLE
Fix issue with domain is null error

### LINKED ISSUE #388: Block library not rendering due to JS error
Screenshot 2023-10-02 at 21 59 19

URL: https://main--bitdefender--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/barchart&index=0

JS error in `getDomainInfo` where `domain` is null.

### PR DESCRIPTION
**Pull request by lansami**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #388 

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/
- After: https://issue-388--bitdefender--hlxsites.hlx.page/solutions/

### CODE REVIEW

**Review by andreibogdan — APPROVED** [review]

File: scripts/breadcrumbs.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export async function loadBreadcrumbs() {
   decorateBlockWithRegionId(breadcrumb, 'Hero|Breadcrumb');
 
   // check if breadcrumb div exists
-  if (breadcrumb !== undefined) {
+  if (breadcrumb !== undefined && breadcrumb !== null) {
```
Comment (basecode) [inline scripts/breadcrumbs.js]: Can we simplify and catch all "nullish" values? -> `if (breadcrumb)`

File: scripts/scripts.js [hunk 2]
```diff
@@ -362,6 +362,10 @@ function getDomainInfo(hostname) {
 
 function pushPageLoadToDataLayer() {
   const { hostname } = window.location;
+  if (!hostname) {
```
Comment (basecode) [inline scripts/scripts.js:365]: I expect the hostname to be not null. Rather `main--bitdefender--hlxsites.hlx.page`?

File: scripts/breadcrumbs.js (hunk 1 above)
Comment (lansami) [reply scripts/breadcrumbs.js]: I reverted this, I saw that there is another PR that has this change

File: scripts/scripts.js (hunk 2 above)
Comment (lansami) [reply scripts/scripts.js:365]: in case of sidekick, when loading the page, the `window.localtion.hostname` is an empty string

Comment (basecode) [reply scripts/scripts.js:365]: ack!

**Pull request merged**